clients/gtest/matmul_gtest.yaml

```diff
@@ -736,7 +734,7 @@ Tests:
   bias_vector: [0, 1]
   bias_type: f32_r
   unit_check: 1
-  gpu_arch: '942'
+  gpu_arch: '94[0-2]'
```

Comment:
Don't need to update 940 nd 941 logic yaml and test.

Reply:
OK, done!!